classroom/snippets/src/main/java/GetTopic.java

```diff
@@ -63,12 +66,7 @@ public static Topic getTopic(String courseId, String topicId) throws IOException
       System.out.printf("Topic '%s' found.\n", topic.getName());
     } catch (GoogleJsonResponseException e) {
       //TODO (developer) - handle error appropriately
-      GoogleJsonError error = e.getDetails();
```

Comment:
why did you modify how these topic errors are handled? I think the Topic error handling sections are different from each other now